spec/services/feature_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe Feature do
```

Comment:
There's already a spec for this; can we consolidate them? https://github.com/DFE-Digital/npq-registration/blob/main/spec/lib/services/feature_spec.rb

Reply:
might be worth improving the general coverage of this service while you're in there; looks like only the `registration_closed?` method is tested at the moment

Reply:
🤦 i will merge it into the correct location.